controller_interface/src/controller_interface.cpp

```diff
@@ -20,148 +20,63 @@
 #include <vector>
 
 #include "hardware_interface/types/lifecycle_state_names.hpp"
-#include "lifecycle_msgs/msg/state.hpp"
 
 namespace controller_interface
 {
 return_type ControllerInterface::init(const std::string & controller_name)
 {
-  node_ = std::make_shared<rclcpp::Node>(
-    controller_name, rclcpp::NodeOptions()
-                       .allow_undeclared_parameters(true)
-                       .automatically_declare_parameters_from_overrides(true));
+  node_ = std::make_shared<rclcpp_lifecycle::LifecycleNode>(
+    controller_name,
+    rclcpp::NodeOptions()
+      .allow_undeclared_parameters(true)
+      .automatically_declare_parameters_from_overrides(true),
+    false);
```

Comment:
Yes, but I am not sure if that is very important. This is something we will not change/use much. We set it up once and then we “forget” about this.

If we would like to enable users to set this value than this would be more than welcome, but since it is “a core functionality of the framework” then I don't think it is necessary.

Reply:
> We set it up once and then we “forget” about this.
yeah exactly the point! :D 
we could also just have a comment there so it's easier to check there instead of having to go look in the lifecycle node's codebase (doxygen is outdated ATM) for what that boolean should represent

Reply:
I would then rather go with a comment then with defining a new constant. (If it is not already defined in LifecycleNode - I couldn't find it)

Reply:
```suggestion
    false);  // disable LifecycleNode service interfaces
```